extensions/gitpod/src/extension.ts

```diff
@@ -231,7 +231,8 @@ export async function activate(context: vscode.ExtensionContext) {
 					GITPOD_LCA_API_PORT: String(apiPort),
 					GITPOD_LCA_AUTO_TUNNEL: String(false),
 					GITPOD_LCA_AUTH_REDIRECT_URL: `${vscode.env.uriScheme}://${context.extension.id}${authCompletePath}`,
-					GITPOD_LCA_VERBOSE: String(vscode.workspace.getConfiguration('gitpod').get<boolean>('verbose', false))
+					GITPOD_LCA_VERBOSE: String(vscode.workspace.getConfiguration('gitpod').get<boolean>('verbose', false)),
+					GITPOD_LCA_SSH_IDLE_TIMEOUT: String(vscode.workspace.getConfiguration('gitpod').get<string>('ssh-idle-timeout', '3h'))
```

Comment:
it should be `sshIdleTimeout`, using dash separated names is alien in VS Code settings

Reply:
updated

Reply:
i think we can go back to `timeout` and merge https://github.com/gitpod-io/gitpod/pull/7372 instead

sorry for back and forth

Reply:
updated, use `timeout` instead